python/lsst/daf/butler/configs/datastores/formatters.yaml

```diff
@@ -13,6 +13,7 @@ CoaddInputs: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
 VisitInfo: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
 ApCorr: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
 PhotoCalib: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
+FilterLabel: lsst.obs.base.formatters.fitsGeneric.FitsGenericFormatter
```

Comment:
It seems to me that serializing in FITS is overkill. A FilterLabel is a couple of key/value pairs isn't it? YAML or JSON seems fine for that. We could fairly trivially make the FilterFormatter work for FilterLabel and Filter since Formatters know the pytype they are formatting.

Reply:
Every component of `ExposureInfo`, now including `FilterLabel`, is held, managed, and persisted using the same framework; this makes the code much simpler and lets us avoid (or, at least, phase out) a lot of the special-case code of the old `ExposureInfo`. Given that `FilterLabel`/`ExposureInfo` already provides a FITS format in order to benefit from these advantages, why should we create a second format instead of reusing one that is supported out of the box?

In my view, the custom `FilterFormatter` was a necessary evil that standardization has made unnecessary.